### DISCUSSION TITLE
Student Developer Pack

### DISCUSSION THREAD

**Discussion by Godse-07** [community/community#119520] [GitHub Education] [answered]
### Hi everyone,

My Student Developer Pack request is approved but still now I am not getting any update regarding this and Pro membership.
![1](https://github.com/community/community/assets/88872669/8f353c94-95f7-48ac-b6c8-b2ece2de1e36)
My request is to check my issue kindly.

**Comment by samus-aran** [comment, marked as answer]
Hi @Godse-07 👋 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍

**Reply by paruldhariwal** [reply]
@samus-aran I am not able to use my Azure 100 credits even after the Github Student Developer pack has been approved

**Reply by ghost** [reply]
@samus-aran my application is showing approved too but give me access from past 5 days .
![Screenshot 2024-07-14 115802](https://github.com/user-attachments/assets/6e43d08f-1dbe-4d04-a7db-71b95f09e3a0)

**Comment by Godse-07** [comment]
Thanks for this support. I get my Student Developer Pack.
